src/libraries/System.Formats.Tar/tests/TarReader/TarReader.ExtractToFile.Tests.Unix.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public partial class TarReader_ExtractToFile_Tests : TarTestsBase
+    {
+        [Fact]
+        public void ExtractToFile_SpecialFile_Unelevated_Throws()
```

Comment:
How do you know you are unelevated here? Don't we run CI tests as `sudo`?

Reply:
It's my understanding that we only run as sudo when we explicitly wrap the code with RemoteExecutor, and that has been the behavior I've seen in the few tests I've protected with RemoteExecutor.

Is that assumption incorrect?

Reply:
I'm not sure. I've seen other tests be conditional on whether the process is elevated or not. For example:

https://github.com/dotnet/runtime/blob/ab0de531c89f87eca90899a47360fe2cb73e2d02/src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs#L43-L44

https://github.com/dotnet/runtime/blob/ab0de531c89f87eca90899a47360fe2cb73e2d02/src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs#L98-L121

I'm not sure of all the ways our tests are run. So if you are expecting this test to only work in an unelevated process, it might be good to stick that in a condition.

Reply:
BTW this test already existed, I merely renamed the file and the test class.

Reply:
ok - if it is already passing in CI, then we can just keep it.

Reply:
Cool.

BTW the last argument of the `RemoteExecutor.Invoke` method is where I indicate that I want the code to be elevated:

```cs
RemoteExecutor.Invoke(() =>
            {
            // Code to elevate here
            }, new RemoteInvokeOptions { RunAsSudo = true }).Dispose();
```